CONTRIBUTING.md

```diff
@@ -32,18 +32,21 @@ hatch run docs:serve
 hatch run docs:build
 ```
 
-## Publish
+## Releasing
```

Comment:
Feel free to ignore this, but I suppose we could say "Tagging and Releasing".  Same would apply to `sknnr`.  A total nit, though.

Reply:
I merged the `sknnr` PR before I looked at this, so I'll be lazy and just leave it for now to keep them consistent :sweat_smile: